### DISCUSSION TITLE
Create an event where useres can apply for without beeing logged in WP

### DISCUSSION THREAD

**Discussion by Zahackn** [liedekef/events-made-easy#229] [Q&A] [answered]
I want to enable this for users which are not WP users.

Is this possible?

**Comment by liedekef** [comment]
yes

**Reply by Zahackn** [reply]
How? I unchecked everything in the event which should do this...

**Reply by liedekef** [reply]
Uncheck the generic option "By default require WP membership to be able to register?" (which is in fact unchecked by default)
And then per event you have the RSVP option "Require WP membership for booking"

**Reply by Zahackn** [reply]
<img width="1453" alt="Screenshot 2023-07-11 135433" src="https://github.com/liedekef/events-made-easy/assets/139230755/28b3e817-d5b8-4f30-b72b-8fed3cdd5997">

Everything is unchecked

**Reply by liedekef** [reply]
Then show me the site/page where it is not working as it should. Currently you're only showing a screenshot of the settings, not the actual problem.

Edit: and make sure to use #_ADDBOOKINGFORM in the event format, not #_ADDBOOKINGFORM_IF_USER_IN_GROUP or #_ADDBOOKINGFORM_IF_USER_IS_MEMBER_OF

**Reply by Zahackn** [reply]
https://www.ej-reitcoaching.de/events/reitstunde-104/

Where do I set this Option? In settings or in the event itself?

**Reply by liedekef** [reply, marked as answer]
Based on that error: what is the status of the event? If it is private or draft, this message will show too.

**Reply by Zahackn** [reply]
It was draft.....

(faceplam)
